pt-BR/mail/chrome/messenger/messengercompose/pbMacroSearchOverlay.dtd

```diff
@@ -3,7 +3,7 @@
 <!ENTITY insertMacro.key "l">
 <!ENTITY insertMacro.modifiers "accel">
 
-<!ENTITY insertMacro.label "Quick Bar">
+<!ENTITY insertMacro.label "Barra rápida">
 
-<!ENTITY pbMacroSearchPanel.firstrunWin "When typing a URL, Postbox will autocomplete against the URLs from your emails. To disable URL autocompletion, go to Options > Composition > Advanced">
-<!ENTITY pbMacroSearchPanel.firstrunMac "When typing a URL, Postbox will autocomplete against the URLs from your emails. To disable URL autocompletion, go to Preferences > Composition > Advanced">
+<!ENTITY pbMacroSearchPanel.firstrunWin "Ao digitar uma URL, o Postbox preencherá autmaticamente a partir das URLs dos seus e-mails. Para desabilitar a funcionalidade de preenchimento automático de URL, vá para Opções > Edição > Avançado.">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY pbMacroSearchPanel.firstrunWin "Ao digitar um URL, o Postbox preencherá autmaticamente a partir das URLs dos seus e-mails. Para desabilitar a funcionalidade de preenchimento automático de URL, vá para Opções > Edição > Avançado.">